Source/Scene/Model.js

```diff
@@ -898,11 +898,16 @@ define([
         var buffers = model.gltf.buffers;
         for (var name in buffers) {
             if (buffers.hasOwnProperty(name)) {
-                ++model._loadResources.pendingBufferLoads;
                 var buffer = buffers[name];
-                var uri = new Uri(buffer.uri);
-                var bufferPath = uri.resolve(model._baseUri).toString();
-                loadArrayBuffer(bufferPath).then(bufferLoad(model, name)).otherwise(getFailedLoadFunction(model, 'buffer', bufferPath));
+
+                if (buffer.type === 'arraybuffer') {
+                    ++model._loadResources.pendingBufferLoads;
+                    var uri = new Uri(buffer.uri);
+                    var bufferPath = uri.resolve(model._baseUri).toString();
+                    loadArrayBuffer(bufferPath).then(bufferLoad(model, name)).otherwise(getFailedLoadFunction(model, 'buffer', bufferPath));
+                } else if (buffer.type === 'text') {
+                    // GLTF_SPEC: Load compressed .bin with loadText.  https://github.com/KhronosGroup/glTF/issues/230
```

Comment:
Is this if block intentionally empty, or is it a TODO?  If so we should remove the comment and write an issue instead.